microsoft-edge/devtools-guide-chromium/whats-new/2023/07/devtools-115.md

```diff
@@ -0,0 +1,177 @@
+a Microsoft account (MSA) that's a work account
+-->
+
+Additionally, you can now connect to multiple symbol servers, in case you need to fetch sourcemaps from different Azure DevOps organizations. You can also filter which sourcemaps you want DevTools to fetch from the symbol server, by specifying the sourcemap URLs in an inclusion or exclusion list.
+
+![Connecting to symbol server using Azure Active Directory from DevTools settings](./devtools-115-images/aad-symbol-server.png)
+
+The message **An Azure DevOps organization must be selected** means double-click the organization, or click it and then press **Enter**.
+
+
+<!-- ------------------------------ -->
+#### Wildcards
+
+In the **Filter behavior** text box, the following wildcards are supported:
+
+| Wildcard | Meaning |
+|---|---|
+| ? | Matches a single character. |
+| * | Matches one or more of any character. |
+
+If you select **Exclusion List**, DevTools attempts to look up any source maps for scripts except those that have a URL that matches one of the entries in the list.
+
+If you select **Inclusion List**, DevTools only attempts to look up source maps for scripts that have URLs that match one of the entries in the list.  For example, suppose you select **Inclusion List** in the **Filter behavior** dropdown list, and then enter the following in the **Filter behavior** text box:
+
+```http
+https://cdn.contoso.com/*
+https://packages.contoso.com/*
+```
+
+In this example, DevTools only attempts to resolve source maps that match these two URL patterns, and doesn't attempt to load source maps for other scripts.
+
+
+<!-- ------------------------------ -->
+#### Source Maps Monitor tool
+
+The [Source Maps Monitor tool](../../../source-maps-monitor/source-maps-monitor-tool.md) has also been updated, to show which URL lookups have or haven't been attempted.  For example:
+
+1. In DevTools **Settings > Symbol Server**, in the **Authorization mode** dropdown list, select **Azure Active Directory**.
+
+1. Double-click the **microsoft** org to select it.
+
+1. In the **Filter behavior** dropdown list, select **Exclusion List**.
+
+1. In the **Filter behavior** text box, enter a domain with wildcard (`https://ryanseddon.com/*`) to exclude.
+
+1. Close DevTools **Settings**.
+
+As a result, the **Source Maps Monitor** tool indicates excluded source map URL lookup attempts:
+
+![Source Maps Monitor tool indicates source map URL lookups attempted](./devtools-115-images/source-maps-monitor.png)
+
+See also:
+* [Securely debug original code by publishing source maps to the Azure Artifacts symbol server](../../../javascript/publish-source-maps-to-azure.md)
+* [Securely debug original code by using Azure Artifacts symbol server source maps](../../../javascript/consume-source-maps-from-azure.md)
+
+
+<!-- ====================================================================== -->
+## Improvements for the Memory tool
```

Comment:
This title is too similar to the one immediately preceding it. Can we come up with a different word for "Improvements"
 ```suggestion
## Allocation instrumentation on timeline works better and heap snapshots are now more accurate in the Memory tool
```

Reply:
`## Memory tool enhancements`